Packs/Intezer/Integrations/IntezerV2/README.md

```diff
@@ -688,3 +698,94 @@ size_in_bytes	        245760
 ssdeep	                ssdeep
 ```
 
+### intezer-get-analysis-iocs
+
+***
+Gets the list of network and files IOCs of a specific analysis id.
+
+#### Base Command
+
+`intezer-get-analysis-iocs`
+
+#### Input
+
+| **Argument Name**   | **Description**                              | **Required**   |
+|---------------------|----------------------------------------------|----------------|
+| analysis_id         | The analysis ID we want to get the IOCs for. | Required       | 
+
+#### Context Output
+
+| **Path**                              | **Type**  | **Description**          |
+|---------------------------------------|-----------|--------------------------|
+| Intezer.Analysis.ID                   | string    | The composed analysis ID | 
+| Intezer.Analysis.IOCs                 | Unknown   | The Analysis IOCs        | 
+
+#### Context Example
+
+```json
```

Comment:
Please verify that all the IPs and file hashes here are not secrets - would be better to use something like 1.1.1.1. 

In addition, I think the example could be shorter - there is no need to add to json more than one file example and one network example.

Reply:
I removed IP but kept network IOCs of two types: IP and domain